user/example/misc/scan/ExampleScan2.scan

```diff
@@ -0,0 +1,40 @@
+[global]
+repeatScans = 0
+allowNested = 1 # overrides local nested arguments
+#configPrefix = "path/to/folder/scanned" // optional
+timeBasedScan = 1 # 1 = true :)
+timePerStep = 10 #in seconds for now
+nEventsPerStep   = 200
+
+[0]
+scanParallelTo = 0
```

Comment:
Why would I like to scan that in parallel to itself? Looks odd to me. What if it is not defined at all?

Reply:
I am not sure what you mean by "that in parallel to itself" if it is about the fact that it affects the same producer this is simply to show that you can also change two variables of the same producer at the same time.

If scanParallelTo is not defined it simply counts each [X] variable as its own unit to ensure backwards compatibility with old scan files.

Reply:
got it